src/App.css

```diff
@@ -13,6 +13,65 @@ html, body, #root, #app {
   font-size: 16px;
 }
 
+.toolbar-container {
+  text-align: center;
+  position: fixed;
+  top: 10px;
+  right: 10px;
+  z-index: 2;
+  -webkit-touch-callout:none;
+  -webkit-user-select:none;
+  -khtml-user-select:none;
+  -moz-user-select:none;
+  -ms-user-select:none;
+  user-select:none;
```

Comment:
We can simply use the unprefixed property, as `user-select` is [supported by all modern browsers](https://caniuse.com/#feat=user-select-none).

Reply:
Ok, I had a thought about the finger being in the way of the overlay. It's more of a touchscreen than a mobile issue. 
I think it's better to check if the device is touch enabled, although that would still not be perfect given a scenario where a device is touch enabled and has a mouse/pointing device.
Let me know what you think.

Reply:
I think you may have replied to the wrong comment. I agree it is a touchscreen issue, and I am open to your judgment about the best solution.

Reply:
Yes, I replied to the wrong comment, sorry about that.